config/main.py

```diff
@@ -1268,15 +1327,26 @@ def warm_restart_bgp_eoiu(ctx, enable):
 #
 @config.group(cls=AbbreviationGroup)
 @click.pass_context
+@click.option('-n', '--namespace', help='Namespace name', default=None)
 @click.option('-s', '--redis-unix-socket-path', help='unix socket path for redis connection')
-def vlan(ctx, redis_unix_socket_path):
+def vlan(ctx, redis_unix_socket_path, namespace):
     """VLAN-related configuration tasks"""
     kwargs = {}
     if redis_unix_socket_path:
         kwargs['unix_socket_path'] = redis_unix_socket_path
-    config_db = ConfigDBConnector(**kwargs)
+
+    # If multi ASIC platform, check if the namespace entered by user is valid
```

Comment:
duplicate code. make it as a single check function.

Reply:
Updated.